DropwizardHelloWorld/src/main/java/com/example/helloworld/HelloWorldConfiguration.java

```diff
@@ -0,0 +1,36 @@
+package com.example.helloworld;
+
+import io.dropwizard.Configuration;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.hibernate.validator.constraints.NotEmpty;
+
+/**
+ * Created by ccardona on 4/11/17.
+ */
+public class HelloWorldConfiguration extends Configuration {
+    @NotEmpty
+    private String template;
+
+    @NotEmpty
+    private String defaultName = "Stranger";
+
+    @JsonProperty
```

Comment:
Why are you annotating application config's fields as JSON properties? Are you planning to have a resource exposing the configuration as JSON?